browser/src/Editor/NeovimEditor/NeovimEditor.tsx

```diff
@@ -744,7 +744,7 @@ export class NeovimEditor extends Editor implements IEditor {
 
         document.body.ondrop = ev => {
             ev.preventDefault()
-            ev.stopPropagation()
+            // ev.stopPropagation()
```

Comment:
Will this break drag-and-drop on the editor? It might be worth separating this out into a full fix for #2070 - I'm assuming it was probably commented out for testing 😄

Reply:
@bryphe yes apologies I mean to remove it actually it fixes the *explorer* drag and drop issues but not sure of its impact on dragging files onto oni was just trying to see if this was the cause of the regression

Reply:
Cool, that's what I figured! Ya, in that case, I think it makes sense to remove it for now, just so we can evaluate the impact on dragging files into the editor as part of a complete fix for #2070